.github/workflows/scripts/build-ispc.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+echo PATH=$PATH
+mkdir build && cd build
```

Comment:
I think it's a good time to use the cmake commands to avoid calling mkdir and make.

See https://github.com/ispc/ispc/blob/0593138b0091464be9681df539ca692a142cfe75/.github/workflows/linux-benchmarks.yml#L47-L48

Note that it will make it easier to switch to ninja for faster build at a later point.

Reply:
I don't have much experience with `cmake --build`, so I was just doing it the old way.
But probably it worthy try it and see if we get any problems.